packages/pdf-api/src/draft/draft.controller.ts

```diff
@@ -53,18 +53,21 @@ export class DraftController {
       },
     },
   })
+  @Put('users/:userName/drafts/:draftName')
   public async saveDraft(
     @Param('draftName') draftName: string,
     @Param('userName') userName: string,
     @Body() draftParams: object,
   ) {
-    logger.debug('saveDraft', { draftName, userName });
+    logger.debug('saveDraft', { draftName, userName, draftParams });
 
-    await this.draftServie.saveDraft({
+    await this.draftService.saveDraft({
       userName,
       draftName,
       params: draftParams,
     });
+
+    return { success: true };
```

Comment:
generally we only return status messages like this when there is an error, e.g. API could return 400 `{"error": "some request error"}` in response to some bad request. When the API call was successfull, returning 200 or 201 is enough, i.e. getting success status code is enough for client to understand that the operation was successful. 

Sometimes in case of successful POST or PUT we could return the newly created (or updated) resource (draft in our case) details. This is especially useful when e.g. POST operation creates a new record in DB with generated on server ID (no other way for client to get that newly created id except for parsing the response)